frontend/apps/reader/modules/readerlink.lua

```diff
@@ -652,6 +653,24 @@ function ReaderLink:onGoToExternalLink(link_url)
     end
     -- Set up buttons for alternative external link handling methods
     local alt_handlers_buttons = {}
+    table.insert(alt_handlers_buttons, {
+        text = _("Copy"),
+        callback = function()
+            UIManager:close(dialog)
+            Device.input.setClipboardText(link_url)
+        end,
+    })
+    table.insert(alt_handlers_buttons, {
+        text = _("As QR code"),
```

Comment:
The PR title fits better. Here are some alternatives.
```suggestion
        text = _("URL to QR code"),
```

```suggestion
        text = _("Display QR code"),
```

```suggestion
        text = _("Show QR code"),
```

Reply:
Somehow, "**as** QR code" sounds better to my broken english ears :)
These sounds ok:
```
URL as QR code
Display as QR code
As QR code
```
`Show QR code` or `Show as QR code` a bit less.

Reply:
As a native American English speaker, "Show QR Code" sounds the most natural.

Reply:
I'm definitely also on the `Show QR code` train.

Reply:
Maybe shorter "QR code" would be enough? Probably no need for translation in all languages this way. Also is there a length limit on button text (considering small screens)?

Reply:
The next ones `Add to Wallabag` and `Open in browser` are not smaller than `Show QR code`, so I guess it's fine.